frontend/src/App.tsx

```diff
@@ -1,12 +1,18 @@
 import "./App.css";
+import CreateTagComponent from "./Tag/CreateTagComponent";
 import ViewQuestionsComponent from "./question/ViewQuestionsComponent";
 
 function App() {
   return (
     <>
       <h1>Home Screen</h1>
       <ViewQuestionsComponent />
-    </>
+      <br />
```

Comment:
<Br> should not be used for this. Styling should be used. Fine for now, I guess.